config/initializers/github_omniauth.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :github, ENV['github_client_id'], ENV['github_client_secret']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.